airflow/api_fastapi/execution_api/datamodels/xcom.py

```diff
@@ -28,3 +28,10 @@ class XComResponse(BaseModel):
     key: str
     value: Any
     """The returned XCom value in a JSON-compatible format."""
+
+
+class XComValuePayload(BaseModel):
```

Comment:
I noticed that we either use the term "Payload" or "Body" across all endpoints. Do you think it's better we call them "Body" as the majority of the endpoints use that?

Reply:
Oh which one says Body?

Reply:
![image](https://github.com/user-attachments/assets/a00d834c-358e-4765-9f42-02398ef2f829)

Reply:
aah those are for the public API, I was looking for the Execution API which we don't intend for public to consume outside of Task SDKs